client/src/components/form.js

```diff
@@ -1,97 +1,138 @@
 import { useState } from "react";
 
 const Form = (props) => {
-    // Initial student in case that you want to update a new student
-    const {initialStudent = {id: null, 
-                            firstname: "", 
-                            lastname: ""}} = props;
-
-
-    // We're using that initial student as our initial state                       
-    const [student, setStudent] = useState(initialStudent);
-
-    //create functions that handle the event of the user typing into the form
-    const handleNameChange = (event) => {
-        const firstname = event.target.value;
-        setStudent((student) => ({ ...student, firstname }));
+  // Initial book in case that you want to update a new book
+  const {
+    initialBook = {
+      id: null,
+      author_f: "",
+      author_l: "",
+      format: "",
+      owned: "",
+      read: "",
+    },
+  } = props;
 
+  // We're using that initial book as our initial state
+  const [book, setBook] = useState(initialBook);
+  const str2bool = (value) => {
+    if (value && typeof value === "string") {
+      if (value.toLowerCase() === "true") return true;
+      if (value.toLowerCase() === "false") return false;
```

Comment:
a little cleaner to do `if (value.toLowerCase())` and if `(!value.toLowerCase()`

Reply:
Could I also do `return value.toLowerCase()` then?